dev/source/docs/sitl-with-webots.rst

```diff
@@ -0,0 +1,89 @@
+.. _sitl-with-webots:
+
+======================
+Using SITL with Webots
+======================
+
+.. youtube:: c5CJaRH9Pig
+    :width: 100%
+
+
+
+`Webots <https://cyberbotics.com/>`__ is a simulator mainly used for robotics. It is easy to build many vehicles using it. Ardupilot project contains Quadcopter and Tricopters projects have been built specially for this simulator.
+
+
+You can download Webots simulator  from `www.cyberbotics.com <https://www.cyberbotics.com/#download/>`__ . To run Webots just type webots in command line.
+
+
+Running the Examples
+====================
+
+#. Open Webots using command line.
+#. Open world from `World examples <https://github.com/ArduPilot/ardupilot/blob/master/libraries/SITL/examples/Webots/worlds>`__
+#. Compile Robot Controller and WorldInfo physics plugin using Webots GUI.
+#. Press Run on Webots GUI. Now the simulator is running.
+#. Run ardupilot SITL using available scripts in `the examples <https://github.com/ArduPilot/ardupilot/tree/master/libraries/SITL/examples/Webots>`__. 
+
+::
+
+   ./libraries/SITL/examples/Webots/dronePlus.sh
+   or
+   ./Tools/autotest/sim_vehicle.py -v ArduCopter -w --model webots-quad:127.0.0.1:5599 --add-param-file=libraries/SITL/examples/Webots/quadPlus.parm
+
+
+.. warning::
+
+   It is always important to run Webots before SITL ArduPilot, otherwise the socket connection will not establish.
+
+
+Simulation Parameters
+=====================
+
+There are two types of parameters, first type is passed to SITL and the second is configured in Webots.
+
+SITL communicates with Webots using TCP sockets, so SITL should share port numbers with Webots. Usually you don't need to change the default port 5599, but when you run multiple robots you need to use different ports for each simulation instance.
+
+
+Advantage of Webots
+===================
+
+#. Controllers can be written in c, c++, python and matlab.
+#. Alot of sensors exists.
+#. Ability to add custom physics to simulate things such as wind.
+#. Ability to add OpenStreetMap and run simulator in environments very simulator to reality. 
+
+
+How to Connect Your Own World with Ardupilot
+============================================
+
+WorldInfo
+~~~~~~~~~
+
+The following parameters are important to be set in WorldInfo
+
+- basicTimeStep = 1 or 2
+
+- northDirection needs to be updated when using `OpenStreetMap <https://cyberbotics.com/doc/doc/openstreetmap-importer>`__
+
+- physics used to attach physics plugin *"sitl_physics_env"* used to simulate wind and drag in a very simple way.
```

Comment:
this is unclear to me...since the language has two subjects and two verbs....needs to be clear what Worldinfo param is being changed and exactly what its being change to..

Reply:
I applied your comments ... Thank you :)